internal/handlers/commonpath/commonpath.go

```diff
@@ -0,0 +1,118 @@
+// Copyright 2021 FerretDB Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package commonpath contains functions used for path.
+package commonpath
+
+import (
+	"errors"
+	"strconv"
+
+	"github.com/FerretDB/FerretDB/internal/types"
+	"github.com/FerretDB/FerretDB/internal/util/iterator"
+	"github.com/FerretDB/FerretDB/internal/util/lazyerrors"
+)
+
+// FindValuesOpts sets options for FindValues.
+type FindValuesOpts struct {
+	// SearchInArray searches by iterating through the whole array to find documents that contains path key.
```

Comment:
Option does not search anything

Reply:
Updated this, feel free to suggest better wording if you can think of any as well :pray: